src/ontology/mondo-edit.obo

```diff
@@ -433092,6 +433092,19 @@ synonym: "PARDS" EXACT [PMID:25647235, PMID:31236307]
 is_a: MONDO:0006502 ! acute respiratory distress syndrome
 created_by: http://orcid.org/0000-0001-5208-3432
 
+[Term]
+id: MONDO:0100138
+name: X-linked recessive mitochondrial myopathy
+def: "A mitochondrial myopathy caused by defects in the MICOS subunit gene APOO (MIC26). Modelling in yeast and flies demonstrate an inability to insert MICOS complex into the inner mitohondrial membrane. Associated symptoms include, lactic acidosis, cognitive impairment and autistic features." [PMID:32439808]
```

Comment:
mild concern with deviation between ldef and tdef but we see this a lot with MoI defined diseases

Reply:
I'm not sure what you mean here. Is any action needed?